tutorials/evoked/30_eeg_erp.py

```diff
@@ -542,6 +550,7 @@ def print_peak_measures(ch, tmin, tmax, lat, amp):
     ax.plot(lat_roi * 1e3, amp_roi * 1e6, marker='*', color='C6')
     ax.axvspan(*(times[ix] * 1e3), facecolor=colors[ix], alpha=0.3)
     ax.set_xlim(-50, 150)  # Show zoomed in around peak
+tight_layout(fig=fig)
```

Comment:
Instead of a new import, you can just use `fig.tight_layout()` I think

Reply:
Good point! I tried setting a tight layout in the `subplots()` call, let's see if this also works (saves us another line).